controller/webapp/api_spec/openapi.yaml

```diff
@@ -0,0 +1,84 @@
+openapi: 3.1.0
+
+info:
+  title: RAP API OpenAPI Description
+  version: "v1"
+
+paths:
+  /{backend}/backend/status/:
+    get:
+      description: |
+        Get the status (flags) for a specific backend.
+      parameters:
+        - name: backend
+          in: path
+          description: name of backend
+          required: true
+          schema:
+            type: string
+            pattern: ^[A-Za-z0-9].+$
+            enum:
+              - test
+              - tpp
+              - emis
```

Comment:
not sure it's wrong, but seems slightly weird to have `pattern` as well as `enum`

Reply:
Oh this was my (misunderstood) attempt to make it not generate weird unicode. It can go.